tests/decodeoutput.cpp

```diff
@@ -126,7 +126,9 @@ class ColorConvert {
     }
     SharedPtr<VideoFrame> convert(const SharedPtr<VideoFrame>& src)
     {
-
+        if (YAMI_FOURCC_P010 == src->fourcc){
```

Comment:
This will have side effect to DecodeOutputDump. It makes P010 to I420 convert failed in file dump. Although we it not supported yet. Could we just reset the convert when we saw the P010 surface in DecodeOutputMD5